guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,279 @@
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-safe-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-safe-mobile@2x.png"
+   alt-text = "A diagram showing the sequence of recovery scenarios."
+   width = 682
+   height = 316
+%}
+
+This way, the family can access the material if needed, but any tampering would be immediately noticeable.
+
+### Assisted inheritance material
+
+To enable the assisted inheritance route for their children, Alice and Bob provide their lawyer, Edward, with an encrypted copy of the recovery kit. They store the password separately in a bank deposit box. This way, Christina and David can access the funds through legal channels if necessary, without compromising privacy.
+
+- **Edward's office**: Encrypted copy of the recovery kit (without password) in a tamper-evident bag
+- **Bank deposit box**: Recovery kit password in a tamper-evident bag
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-secondary.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-secondary@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-secondary-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-secondary-mobile@2x.png"
+   alt-text = "A diagram showing the sequence of recovery scenarios."
+   width = 751
+   height = 397
+%}
+
+### Privacy considerations
+
+This approach makes a potential trade-off in terms of privacy, because the children could access the backup material and recover the wallet. This is acceptable for Alice and Bob, since they have good relationships and trust their children. After all, this is the reason they have chosen to include them in the setup in the first place.
+
+But even if Christina and David would breach their parents’ trust and import the wallet, they would not be able to move any funds, because their inheritance keys are time locked. And since Alice and Bob access their safe regularly, they would notice that the bag with the backup material has been opened. They can then safely move the funds to a new wallet, for which they would implement more restrictive security measures.
+
+### Inheritance planning
+
+Alice and Bob keep a copy of their Last Will in the home safe. It references the bitcoin wallet and provides a lead to Edward, their lawyer. To ensure a smooth inheritance process, Alice and Bob also provide a sealed copy of the Will to Edward.
+
+## Balancing security and privacy
+
+The setup oulined above addresses some important aspects of wallet backups:
+
+1. **Tamper-evidence**: The tamper-evident bag in the home safe allows Alice and Bob to quickly check if anyone has attempted to access the backup materials
```

Comment:
Vault?
```suggestion
1. **Tamper-evidence**: The tamper-evident bag in the home safe allows Alice and Bob to quickly check if anyone has attempted to access the backup materials
```

Reply:
Here we are talking about the physical safe in Alice's and Bob's home. Vaults refer to digital vaults in the password manager application.